pcu/PCU.cc

```diff
@@ -52,6 +52,23 @@
 #include <time.h>
 namespace pcu {
 
+int PCU_Init(int *argc, char ***argv) {
+#ifndef SCOREC_NO_MPI
+  return MPI_Init(argc, argv);
```

Comment:
We should check if MPI was already initialized here (https://www.mpich.org/static/docs/latest/www3/MPI_Initialized.html).  Initializing multiple times is undefined behavior or 'wrong' (https://docs.open-mpi.org/en/v5.0.x/man-openmpi/man3/MPI_Init.3.html#mpi-init, https://www.mpich.org/static/docs/v3.3/www3/MPI_Init.html).

Reply:
Should pcu::PCU_Init throw an exception in this case?

Reply:
Nope, we just don't call initialize.

Reply:
That works for me.  Thanks for checking.